### ISSUE TITLE
Error is not allowed to connect to this MariaDB server

### ISSUE THREAD

**Issue by deanm1990** [Cyberbyte-Studios/CyberWorks#77]
Hi Guys

Read the other issues on this, but the responses are vague.

If i remember rightly,

When you create a new user for  a remote connection from a web server, the host option needs to be the ip that the database is hosted on ? is that correct ?

I also remember though either the user - or host option had the % symbol in it for somewhere. 

Can someone explain step by step on what exactly needs to be in the username and host option when creating a new user - The privelages are all set.

**Comment by MightySCollins** [comment]
How is your MySQL server currently setup? Though something like XAMPP, cPanel or on a Linux server. You are right about the `%` sign. If you add that the user can connect to the database from any IP although I would recommend using your web server's ip.

Depending on your setup you also might need to bind it to `0.0.0.0`. If you can give me as much detail as possible I might be able to do you a guide.

**Comment by deanm1990** [comment]
Xampp

http://imgur.com/a/zNo5h

So is it just one % symbol in the user slot or do i need %any% or something like that

**Comment by Jason2605** [comment]
If you wish the user to have access from anywhere its just the % on its own, other wise you can specify an ip, or an ip range

**Comment by deanm1990** [comment]
OK , i put just the % in the host section,  still get this though Connect failed: Access denied for user 'AdminPanel'@'ns1.uk3.siteground.eu' (using password: YES)

**Comment by Jason2605** [comment]
And can you connect to the DB externally, say from your computer not where the database is hosted itself, that can be a good identifier to make sure of the permissions.

**Comment by MightySCollins** [comment]
Yep because it seems you have already externally bound it if it can access it

**Comment by deanm1990** [comment]
ok working :) , but no information on players or vehicles

**Comment by Jason2605** [comment]
By no information do you mean the tables aren't populated? Also what version of AL are you running?

**Comment by deanm1990** [comment]
its arma 3 life - i've used this panel with it before with no issues.

But yes the players are not populated. but the main dashboard shows total number of vehicles and players

**Comment by Jason2605** [comment]
Okay, well I'm guessing that was based off of the Altis Life framework still, so instead. In your database in the players table, is it playerid or pid?

**Comment by deanm1990** [comment]
But yes the players are not populated. but the main dashboard shows total number of vehicles and players

**Comment by deanm1990** [comment]
http://imgur.com/a/wPU7u

**Comment by MightySCollins** [comment]
Can you uncomment line 3 from index.php and show us any errors.

**Comment by Jason2605** [comment]
Its due to the settings being set to pid, give me 15 mins, and ill push something. That should be able to fix it

**Comment by MightySCollins** [comment]
Ahh yep I forgot I set that to default. A nice settings option in `firstTime.php` might help.

**Comment by Jason2605** [comment]
Yup, and in the options tab.

**Comment by deanm1990** [comment]
ok :) like it, can you let me know when to download new update

**Comment by deanm1990** [comment]
Im very impressed with the reply responses- will certainly make a donation

**Comment by MightySCollins** [comment]
Thanks :smile: https://paypal.me/CyberbyteStudios ill make sure @Jason2605 gets some too

**Comment by Jason2605** [comment]
Ill try and get this done quite soon, sorry something came up.

**Comment by Jason2605** [comment]
Give me, 20/30 mins

**Comment by Jason2605** [comment]
@deanm1990 Ive pushed the changes, it should now allow the selection between pid(AL 5.0) and playerid (all other vers). Please let me know if you experience any problems with the recent change, and ill try my best to amend them.

**Comment by deanm1990** [comment]
Hey, been ill so just had chance to see the new update. All works now, can see all data etc. The only thing is,  when i make changes on the panel, it doesn't update on the DB, except for licenses, that works ok.
